index.bs

```diff
@@ -469,9 +467,10 @@ apply to the shadow host chain.
 
 <em>This section is non-normative.</em>
 
-The API applies only to {{HTMLVideoElement}} in order to start on a minimal
-viable product that has limited security issues. Later versions of this
-specification may allow PIP-ing arbitrary HTML content.
+To prevent potential abuse through spoofing, the API applies only to
```

Comment:
I don't think we can prevent it... a video can still masquerade as a system ui or an alert, which could confuse users:

```suggestion
To limit potential abuse through spoofing, the API applies only to
```